src/data/external/osmosis.ts

```diff
@@ -0,0 +1,71 @@
+import { useQuery } from "react-query"
+import request from "axios"
+import { queryKey } from "../query"
+
+// https://api-osmosis.imperator.co/swagger/
+export const OSMOSIS_API_URL = "https://api-osmosis.imperator.co"
+
+export const GAMM_TOKEN_DECIMALS = 18
+export const OSMO_ICON =
+  "https://raw.githubusercontent.com/terra-money/station-assets/main/img/chains/Osmosis.svg"
```

Comment:
I would use this image link here: https://station-assets.terra.money/img/chains/Osmosis.svg
It's the same image but from the deployed URL